metricflow/query/validation_rules/metric_time_requirements.py

```diff
@@ -65,6 +46,18 @@ def _group_by_items_include_metric_time(self, query_resolver_input: ResolverInpu
 
         return False
 
+    def _group_by_items_include_agg_time_dimension(
+        self, query_resolver_input: ResolverInputForQuery, metric_reference: MetricReference
```

Comment:
This change was made in the other PR too, right?

Reply:
Right. Planning to resolve that in merge conflict resolution.

Reply:
Next time will probably base the branches off of each other instead... Didn't realize they still had overlap when I made that choice.